src/Marten/Marten.csproj

```diff
@@ -22,17 +22,17 @@
         <PackageReference Include="BaselineTypeDiscovery" Version="1.1.2" />
         <PackageReference Include="LamarCodeGeneration" Version="3.2.2" />
         <PackageReference Include="LamarCompiler" Version="3.0.4" />
-        <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[2.1.0, 6.0.0]" />
+        <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
         <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-        <PackageReference Include="Npgsql.Json.NET" Version="5.0.7" />
+        <PackageReference Include="Npgsql.Json.NET" Version="[5.0.10,6.0)" />
         <PackageReference Include="Remotion.Linq" Version="2.2.0" />
         <PackageReference Include="System.Memory" Version="4.5.4" />
         <PackageReference Include="System.Threading.Tasks.Dataflow" Version="5.0.0" />
         <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[2.1.0, 6.0.0]" />
-        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="[2.1.0, 6.0.0]" />
+        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
+        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="2.2.0" />
```

Comment:
The `*.Abstractions` packages can be moved to `3.1.0`, as anything below that is end of life.

Reply:
I thought about that, but I just followed the SemVer rules (same case for Lamar, etc.). I think that we'll have the v5 release soon because of the Npgsql 6. Maybe then we could upgrade it, thoughts?

Reply:
Yeah that's reasonable 👍